File: 11INFORMATIKA/DATASCIENCE/11KOM2/meet02/meet02i/pacar.py

class Pacar:

    def __init__(self, name, age, sex = "Female"):

        self.name = name
        self.sex = sex
        self.age = age

    def say_hi(self):
        print(f"Hi, saya {self.name}")

    def nongkrong(self):
        print("Ayo ...")


pacar_satu = Pacar("Tania", 15, "Female")
pacar_satu.say_hi()
pacar_satu.nongkrong()

pacar_dua = Pacar("Sylvi", 18)
pacar_dua.say_hi()

File: 11INFORMATIKA/DATASCIENCE/11KOM2/meet18/pacar.cpp
#include <iostream>

struct Pacar {
    std::string name;
    int age;
    bool sex;

    Pacar(std::string n, int a) : name(n), age(a), sex(false){}
    Pacar(std::string n, int a, bool s) : name(n), age(a), sex(s){}

    void say_hi()
    {

    }

    void nongkrong()
    {

    }
};

int main()
{
    Pacar pacar_satu("Tania", 15, false);
    pacar_satu.say_hi();
    pacar_satu.nongkrong();

    Pacar pacar_dua("Sylvi", 18);
    pacar_dua.say_hi();
    pacar_dua.nongkrong();
    return 0;
}